src/room/InCallView.tsx

```diff
@@ -442,53 +424,63 @@ export function InCallView({
         {renderContent()}
         {footer}
       </div>
-      {otelGroupCallMembership && (
+      {/*otelGroupCallMembership && (
         <GroupCallInspector
           client={client}
           groupCall={groupCall}
           otelGroupCallMembership={otelGroupCallMembership}
           show={showInspector}
         />
-      )}
+      )*/}
       {rageshakeRequestModalState.isOpen && !noControls && (
         <RageshakeRequestModal
           {...rageshakeRequestModalProps}
-          roomId={groupCall.room.roomId}
+          roomId={rtcSession.room.roomId}
         />
       )}
       {settingsModalState.isOpen && (
         <SettingsModal
           client={client}
-          roomId={groupCall.room.roomId}
+          roomId={rtcSession.room.roomId}
           {...settingsModalProps}
         />
       )}
       {inviteModalState.isOpen && (
-        <InviteModal roomId={groupCall.room.roomId} {...inviteModalProps} />
+        <InviteModal roomId={rtcSession.room.roomId} {...inviteModalProps} />
       )}
     </div>
   );
 }
 
+function findMatrixMember(
+  room: MatrixRoom,
+  id: string
+): RoomMember | undefined {
+  if (!id) return undefined;
+
+  const parts = id.split(":");
+  if (parts.length < 2) {
```

Comment:
Should this be `3`, if the ID looks like `@simonb:element.io:FDU546DFE`?

Reply:
I don't think so? There should be at least two parts: the user ID and the device ID. If there are fewer than that, it's an error.

Reply:
Oh, I see what you mean. Well, this was not assuming anything in particular about the format of the user IDs other than that they might contain colons, so we just take the last part off the end. Really we ought to pick a separator that doesn't appear in either, escaping/encoding the parts if necessary.

Reply:
I agree, we should use something different and that the code that joins the two user ID parts works but this check doesn't...

Reply:
Fair enough, I guess we specifically know it's a user ID anyway so we should treat it as such.